src/pages/PingDetail.svelte

```diff
@@ -5,8 +5,52 @@
   const pingDataPromise = getPingData(params.app, params.ping);
 </script>
 
+<style>
+  .table-header {
+    @apply table-auto;
+    @apply my-4;
+  }
+
+  .table-header td {
+    @apply border;
+    @apply p-2;
+  }
+</style>
+
 {#await pingDataPromise then ping}
   <h1>{ping.name}</h1>
-  <p>{ping.description}</p>
+  <table class="bg-gray-100 table-header">
```

Comment:
Let's skip the styling for now.

```suggestion
  <table class="table-header">
```